java/test/jmri/jmrit/logixng/tools/ImportEntryExitTest.java

```diff
@@ -49,26 +49,14 @@ private void deleteAllLogixs() {
         }
     }
 
-    private boolean isEnabled(DestinationPoints dp) {
-//        Method retrieveItems = dp.getClass().getDeclaredMethod("isEnabled", String.class);
-        try {
-            Method isEnabled = dp.getClass().getDeclaredMethod("isEnabled");
-            isEnabled.setAccessible(true);
-            return (boolean)isEnabled.invoke(dp);
-        }
-        catch ( IllegalAccessException | IllegalArgumentException | NoSuchMethodException | SecurityException | InvocationTargetException ex ){
-            return false;
-        }
-    }
-
     private void runTestActionEntryExit(DestinationPoints dp, Sensor sensor) throws JmriException, NoSuchMethodException, IllegalAccessException, IllegalArgumentException, InvocationTargetException {
```

Comment:
No need to change, but the exceptions NoSuchMethodException, IllegalAccessException, IllegalArgumentException, InvocationTargetException are no longer thrown.